src/typegoose.providers.spec.ts

```diff
@@ -24,27 +24,31 @@ describe('createTypegooseProviders', () => {
 
   beforeAll(async () => {
     jest.setTimeout(120000);
-    await mockgoose.prepareStorage();
-
-    connection = await mongoose.createConnection('mongodb://foobar/baz');
-    setTimeout(() => {
-      connection.close(err => {
-        if (err) return console.log(err);
-        console.log('disconnected');
-      });
-    }, 60000);
 
+    connection = await mongoose.createConnection(await mongod.getConnectionString(), {
+      useCreateIndex: true,
+      useFindAndModify: true,
+      useNewUrlParser: true,
+      useUnifiedTopology: true
+    });
+    // setTimeout(() => {
```

Comment:
before merging i wanted to ask why this was here, and if it could be removed

Reply:
I was getting deprecation warnings from mongoose regarding schema options. I followed that they recommended at the time and added these options.

Reply:
> I followed that they recommended at the time and added these options.

sorry if github was not showing this correctly, but i meant the commented-out `setTimeout` block

Reply:
Yeah, it can be removed.

Reply:
removed in 4de39a3ef984669f0d3b65118fb7e2283e2d7fb9